agent/lib/kontena/websocket_client.rb

```diff
@@ -171,6 +172,7 @@ def on_close(event)
         handle_invalid_version
       end
       info "connection closed with code #{event.code}"
+      notify_actors('websocket:close', nil)
```

Comment:
You mean have both of these cases notify `websocket:close`? I suppose, but then it would just sometimes get two `websocket:close` notifications when disconnecting. Now it might get a `websocket:disconnect`, and then finally it will always get a `websocket:close`.

The only thing subscribing to these notifications is the `LogWorker`.